### PR TITLE
Landing page ps81

### PR DESCRIPTION
**Pull request by kpodemski**
PS 8.1 landing page.

### CODE REVIEW

File: content/releases/prestashop81.md [hunk 1]
```diff
@@ -0,0 +1,481 @@
+---
+title: What's New in PrestaShop 8.1
+menu:
+  main:
+    parent: "Download"
+    name: "What's new"
+showIntroductionBanner: false
+
+subtitle: 
+summary: PrestaShop 8 is a major update that brings an updated Symfony 4.4 version, compatibility with PHP 8.1, new password policy and session management features, support for WebP, and more.
```
Comment (eternoendless) [inline content/releases/prestashop81.md:10]: Don't forget to update this summary ;)

File: content/releases/prestashop81.md [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+                            <p>
+                                In 8.1 you can efficiently manage your products across multiple stores with our improved multistore features. Assign products to a specific store, access other stores not associated with the product, or manage all stores with ease. This approach helps avoid errors and ensures secure product assignments. To associate a product with a new store, simply select it in the multistore header, eliminating the risk of accidentally creating products in new shops without your knowledge.
+                            </p>
+                            <p>
+                                The new "Apply changes to all stores" checkbox offers flexibility by impacting content only on the selected input of the associated stores. This safer, more controlled approach enables you to manage changes across your stores with precision.</p>
+                            <p>
+                                To associate a product with a new store, simply select it in the multistore header. This secure method ensures that no products are accidentally created in a new shop without your knowledge.
+                            </p>
```
Comment (eternoendless) [inline content/releases/prestashop81.md:253]: Is it multistore, multi store, or multi-store?
```suggestion
                            <p>
                                Assigning products to a specific store, understanding when you are accessing products not associated with the current shop, and managing products across multiple stores has been made easier and more clear.
                            </p>
                            <p>
                                A dedicated checkbox allows to choose whether to apply changes to the current store or to all stores. Assigning a product to a store is now done through the new multistore header, ensuring that no products are accidentally created in a new store accidentally.
                            </p>
```

File: content/releases/prestashop81.md [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+  _defaults:
+    autowire: true
+    bind:
+      $elements: !tagged test_module.instance_of.instance_of_tagged
+
+  _instanceof:
+    TestModule\InstanceofConditionals\Collection\ElementInterface:
+      tags: [ test_module.instance_of.instance_of_tagged ]{{< / highlight >}}
+                            </p>
+                            <p>Check the <a href="https://devdocs.prestashop-project.org/8/modules/concepts/services/#advanced-services-parameters-_instanceof-or-interface-binding-manual-tags">developer documentation</a> for more infromation</p>
```
Comment (coffrep) [inline content/releases/prestashop81.md:443]:
```suggestion
                            <p>Check the <a href="https://devdocs.prestashop-project.org/8/modules/concepts/services/#advanced-services-parameters-_instanceof-or-interface-binding-manual-tags">developer documentation</a> for more information</p>
```

**Review by coffrep — DISMISSED** [review]
Hi Krystian, great page for this new version, looks fantastic. Just a few suggestions.

**Pull request merged**